website/docs/language/state/sensitive-data.mdx

```diff
@@ -35,3 +35,6 @@ For example:
 - The S3 backend supports encryption at rest when the `encrypt` option is
   enabled. IAM policies and logging can be used to identify any invalid access.
   Requests for the state go over a TLS connection.
+- The GCS (Google Cloud Storage) backend supports:
```

Comment:
How would you feel about combining these into one more concise sentence for readability? I also suggest changing the link to be relative links:
```suggestion
- The GCS (Google Cloud Storage) backend supports using [customer-supplied](/terraform/language/settings/backends/gcs#customer-supplied-encryption-keys) or [customer-managed (Cloud KMS)](/terraform/language/settings/backends/gcs#customer-managed-encryption-keys-cloud-kms) encryption keys.
```

Reply:
Sounds good to me, thanks!